smart_short_generator.py

```
@@ -614,6 +611,60 @@ def _calculate_risk_reward(self, entry_price: float, df: pd.DataFrame, index: in
             logger.error(f"Грешка при risk/reward calculation: {e}")
             return 0
 
+    def _should_allow_short_signals(self, market_regime: Dict[str, Any]) -> bool:
+        """
+        КРИТИЧНА ЛОГИКА: Определя дали SHORT сигнали са позволени
```

Ruff (0.12.2)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)